src/support/action/acceptCookieBanner.js

```diff
@@ -0,0 +1,13 @@
+// To dismiss cookie banner, as it blocks clicks
+
+module.exports = (text) => {
+
+	browser.pause(2000);
+
+	const cookieControl = $("body #ccc");
+		if (cookieControl.isVisible("button=Accept all cookies")) {
```

Comment:
The indentation seems a bit off here - can you align pls?

Reply:
aligned, please check this is right please.